bigbluebutton-html5/imports/ui/components/recording/container.jsx

```diff
@@ -19,5 +19,6 @@ export default withModalMounter(withTracker(({ mountModal }) => ({
   },
 
   recordingStatus: (Meetings.findOne({ meetingId: Auth.meetingID }).recordProp.recording),
+  recordingTime: (Meetings.findOne({ meetingId: Auth.meetingID }).recordProp.time),
```

Comment:
Is it possible to combine the lookup for the recording status and time? You're finding and using the exact same property of the Meeting for both.